File: app/src/main/java/me/magnum/melonds/ui/input/IInputListener.java
package me.magnum.melonds.ui.input;

import me.magnum.melonds.model.Input;
import me.magnum.melonds.model.Point;

public interface IInputListener {
	void onKeyPress(Input key);
	void onKeyReleased(Input key);
	void onTouch(Point point);
}

File: app/src/main/java/me/magnum/melonds/ui/input/IInputListener.kt
package me.magnum.melonds.ui.input

import me.magnum.melonds.domain.model.Input
import me.magnum.melonds.domain.model.Point

interface IInputListener {
    fun onKeyPress(key: Input)
    fun onKeyReleased(key: Input)
    fun onTouch(point: Point)
}